ChIP-Seq/tutorials/TAL1_binding_site_identification.md

```diff
@@ -0,0 +1,420 @@
+
+    | :grey_question: Questions |
+    |:---|
+    | <ul><li>What do you see?</li><li>What gene(s) other than Runx1 could be regulated by Tal1?</li></ul> |
+    
+
+
+    <img src="../images/Trackster_Runx1_locus.png"/>   
+    <figcaption><b>Figure X:</b> The Runx1 locus.</figcaption>
+
+<a name="step9"/></a>
+### Step 9: Identifying unique and common Tal1 peaks between the G1E and megakaryocyte states
+
+We've just processed ChIP-seq data from two stages of hematopoiesis and have lists of Tal1 occupied sites (peaks) in both cellular states. Now let's identify Tal1 peaks that are shared between the two cellular states and also those that are specific to one cellular state.
+
+:pencil2: ***Hands on!***
+
+1. Run the tool `bedIntersect` to find peaks that exist both in G1E and megakaryocytes. 
+
+  - Select the "Tal1 G1E peaks" and "Tal1 Mega peaks" files as the inputs. Running this tool with the default settings will return overlapping peaks of both files. 
+  
+2. Run the tool `bedIntersect` to find peaks that exist only in G1E.
+
+  - Select "Tal1 G1E peaks" as the first input and "Tal1 Mega peaks" as the second input file.
+  - **Report only those alignments that \*\*do not\*\* overlap the BED file**: Yes
+  
+3. Run the tool `bedIntersect` to find peaks that exist only in megakaryocytes.
+
+  - Select "Tal1 Mega peaks" as the first input and "Tal1 G1E peaks" as the second input file.
+  - **Report only those alignments that \*\*do not\*\* overlap the BED file**: Yes  
+  
+4. Re-name the three files we generated to reflect their contents. 
+
+    | :grey_question: Questions |
+    |:---|
+    | <ul><li>How many Tal1 binding sites are common to both G1E cells and megakaryocytes?</li><li>How many are unique to G1E cells?</li><li>How many are unique to megakaryocytes?</li></ul> |
+
+<a name="stepA"/></a>
+# OPTIONAL ADDITIONAL ANALYSES
+
+### Additional analyses in progress
+
+<a name="conclusion"/></a>
+# Conclusion
+
+In this exercise you imported raw Illumina sequencing data, evaluated the quality before and after you trimmed reads with low confidence scores, algined the trimmed reads, identified Tal1 peaks relative to the negative control (background), and visualized the aligned reads and Tal1 peaks relative to gene structures and positions. Additional, you assessed the "goodness" of the experiments by looking at metrics such as GC bias and IP enrichment. 
+
+:grey_exclamation: ***Key Points***
+
+- *Sophisticated analysis of ChIP-seq data is possible using tools hosted by Galaxy.*
+- *Genomic dataset analyses require multiple methods of quality assessment to ensure that the data are appropriate for answering the biology question of interest.*
+- *By using the sharable and transparent Galaxy platform, data analyses can easily be shared and reproduced.*
+
+## :clap: Congratulations on successfully completing this tutorial!
+
+<a name="literature"/></a>
+# Useful literature
+
+### ChIP-seq
```

Comment:
I started to collect the useful literature for Chip-Seq in the Chip-Seq README file, to limit redundancy of literature if there is several tutorials. What do you think?